seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/ArrayType.java

```diff
@@ -51,6 +51,10 @@ public class ArrayType<T, E> implements SeaTunnelDataType<T> {
     public static final ArrayType<LocalTimeType[], LocalTimeType> OFFSET_DATE_TIME_ARRAY_TYPE =
             new ArrayType(LocalTimeType[].class, LocalTimeType.OFFSET_DATE_TIME_TYPE);
 
+    public static final ArrayType<MapType[], MapType> MAP_ARRAY_TYPE =
```

Comment:
I think the map array should be the same as the decimal type. We cannot define a universal static type, but it should be defined dynamically.